### PR TITLE
Feat/talent

### PR DESCRIPTION
**Pull request by blackicon-eth**
- Migrated database to add a new WaitlistRequirementsType
- Added the new requirement in the update and create waitlist endpoints
- Added a check in the "join" endpoint to verify the user has the minimum required score

### CODE REVIEW

File: app/api/waitlists/[id]/route.ts [hunk 1]
```diff
@@ -228,5 +243,15 @@ export const DELETE = async (
...
   }
+
+  // delete waitlist requirements
+  await prisma.waitlistRequirement.deleteMany({
+    where: {
+      waitlistId: waitlist.id,
+    },
+  });
+
+  // Maybe we also need to delete waitlisted users?
```
Comment (limone-eth) [inline app/api/waitlists/[id]/route.ts:254]: Good catch, but will do in another PR

File: app/frames/waitlists/[slug]/join/route.tsx [hunk 2]
```diff
@@ -320,10 +324,50 @@ const frameHandler = frames(async (ctx) => {
         };
       }
     }
+    if (requiredBuilderScore) {
+      const talentPassportProfile = await getTalentPassportByWallet(
+        ctx.message.requesterVerifiedAddresses[0] ??
```
Comment (limone-eth) [inline app/frames/waitlists/[slug]/join/route.tsx:329]: Shouldn't we use userToAdd.address?

Comment (blackicon-eth) [reply app/frames/waitlists/[slug]/join/route.tsx:329]: Oh god yes, indeed.
I was tired and my brain farted lol

File: app/api/waitlists/[id]/route.ts (hunk 1 above)
Comment (blackicon-eth) [reply app/api/waitlists/[id]/route.ts:254]: Ok, I'll remove it 💯

**Pull request merged**